### PR TITLE
feat(kuma-cp) add versions endpoint

### PR DESCRIPTION
**Pull request by bartsmykla**
### Summary

Added `/versions` endpoint

The versions which are present in a json file, were taken from envoys included in every release starting from `1.0.0` by looking into Dockerfile of our image containing kuma-dp (i.e. https://github.com/kumahq/kuma/blob/1.0.0/tools/releases/dockerfiles/Dockerfile.kuma-dp#L2)

ref.
* Warnings using this endpoint in GUI: https://github.com/kumahq/kuma/pull/1563

### Documentation

- [ ] Link to the website [documentation PR](https://github.com/kumahq/kuma-website/pull/XXX)

### CODE REVIEW

File: pkg/api-server/versions_ws_test.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+		Expect(data).ToNot(BeNil())
+		Expect(data.KumaDp).ToNot(BeNil())
+		Expect(data.KumaDp["1.0.7"]).ToNot(BeNil())
+		Expect(data.KumaDp["1.0.7"].Envoy).To(Equal("1.16.2"))
+
+		// 1.0.8
+		Expect(data).ToNot(BeNil())
+		Expect(data.KumaDp).ToNot(BeNil())
+		Expect(data.KumaDp["1.0.8"]).ToNot(BeNil())
+		Expect(data.KumaDp["1.0.8"].Envoy).To(Equal("1.16.2"))
```
Comment (nickolaev) [inline pkg/api-server/versions_ws_test.go:103]: I don't think there is a need to check each and every entry ;) Maybe just first,middle,last?

**Review by nickolaev — COMMENTED** [review]
Hmm question would be, how to maintain this json "automagically" at every release?

**Review by jakubdyszkiewicz — CHANGES_REQUESTED** [review]
I'm pretty sure this won't work on prod. You need to embed this JSON into binary.

**Review by jakubdyszkiewicz — COMMENTED** [review]
it's probably better to just embed this into go as a string

File: pkg/api-server/versions_ws_test.go (hunk 1 above)
Comment (bartsmykla) [reply pkg/api-server/versions_ws_test.go:102]: I'm checking it as an additional layer for us to be sure we are presenting expected versions

**Comment by bartsmykla** [comment]
@jakubdyszkiewicz as you suggested, embeded it

File: pkg/api-server/versions_ws.go [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    "1.0.7": {
+      "envoy": "1.16.2"
+    },
+    "1.0.8": {
+      "envoy": "1.16.2"
+    }
+  }
+}`)
+
+func versionsWs() (*restful.WebService, error) {
```
Comment (jakubdyszkiewicz) [inline pkg/api-server/versions_ws.go:39]: error is unnecessary

**Review by jakubdyszkiewicz — APPROVED** [review]

File: pkg/api-server/versions_ws.go (hunk 2 above)
Comment (bartsmykla) [reply pkg/api-server/versions_ws.go:39]: great catch, removing

Comment (bartsmykla) [reply pkg/api-server/versions_ws.go]: done

**Pull request merged**